pyproject.toml

```diff
@@ -34,4 +34,40 @@ exclude = ["weaviate/proto", "docs", "integration_v3"]
 
 [[tool.mypy.overrides]]
 module = "weaviate.proto.v1.*"
-ignore_errors = true
\ No newline at end of file
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "grpc.*"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "grpc"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "weaviate_agents.*"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "weaviate_agents"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "weaviate.agents.*"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "weaviate.agents"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "weaviate.__init__"
+ignore_errors = true
```

Comment:
This looks like it might ignore all type errors originating in the `weaviate` package when it is used in tests, is that desired?